cardano-cli/test/Test/Golden/TxView.hs

```diff
@@ -53,7 +53,39 @@ golden_view_shelley :: Property
 golden_view_shelley =
   propertyOnce $
   moduleWorkspace "tmp" $ \tempDir -> do
-    transactionBodyFile <- noteTempFile tempDir "transaction-body-file"
+    updateProposalFile <- noteTempFile tempDir "update-proposal"
+    transactionBodyFile <- noteTempFile tempDir "transaction-body"
+
+    -- Create update proposal
+    void $
+      execCardanoCLI
+        [ "governance", "create-update-proposal"
+        , "--decentralization-parameter", "63/64"
+        , "--epoch", "64"
+        , "--extra-entropy", "c0ffee"
+          --  hash(c0ffee) =
```

Comment:
So rather than leaving comments with the hashes we should output this in the logs using `note_` from `Hedgehog.Extras.Test`.

Reply:
done